drawBot/context/tools/variation.py

```diff
@@ -42,3 +44,45 @@ def getVariationAxesForFontName(fontName):
         data = dict(name=name, minValue=minValue, maxValue=maxValue, defaultValue=defaultValue)
         axes[tag] = data
     return axes
+
+
+@memoize
+def getVariationNamedInstancesForFontName(fontName):
+    """
+    Return a dict { postscriptName: location } of all named instances in a given font.
+    """
+    instances = {}
+    font = CoreText.CTFontCreateWithName(fontName, 12, None)
+    if font is None:
+        return instances
+    cgFont = CoreText.CGFontCreateWithFontName(fontName)
```

Comment:
`cgFont` is only used in the `if "fvar" in ft` code path, better move it there

Reply:
this is solved